content/en/tracing/setup_overview/compatibility_requirements/php.md

```diff
@@ -44,31 +44,37 @@ PHP APM supports the following SAPI's:
-| Symfony 4      | 4.x           | All supported PHP versions |
-| Symfony 5      | 5.x (as of `0.50.0`) | All supported PHP versions |
-| WordPress      | 4.x, 5.x      | PHP 7+                     |
-| Zend Framework | 1.12          | All supported PHP versions |
-| Yii            | 1.1, 2.0      | All supported PHP versions |
-| Drupal         |               | All supported PHP versions |
-| Magento        | 1, 2          | All supported PHP versions |
-| Phalcon        | 1.3, 3.4      | All supported PHP versions |
-| Slim           | 2.x, 3.x, 4.x | All supported PHP versions |
-| Neos Flow      | 1.1           | All supported PHP versions |
-| FuelPHP        | 1.1           | PHP 7+                     |
+**Web frameworks with internals tracing**:
+
+| Module         | Versions             | Support Type               | Instrumentation level           |
+|:-------------- |:---------------------|:---------------------------|:--------------------------------|
+| CakePHP        | 2.x                  | All supported PHP versions | Framework-Level instrumentation |
+| CodeIgniter    | 2.x                  | PHP 7+                     | Framework-Level instrumentation |
+| CodeIgniter    | 3.x                  | PHP 7+                     | Generic web tracing             |
+| Laravel        | 4.2, 5.x, 6.x        | All supported PHP versions | Framework-Level instrumentation |
+| Laravel 8      | 8.x (as of `0.52.0`) | All supported PHP versions | Framework-Level instrumentation |
+| Lumen          | 5.2+                 | All supported PHP versions | Framework-Level instrumentation |
+| Symfony 3      | 3.3, 3.4             | All supported PHP versions | Framework-Level instrumentation |
+| Symfony 4      | 4.x                  | All supported PHP versions | Framework-Level instrumentation |
+| Symfony 5      | 5.x (as of `0.50.0`) | All supported PHP versions | Framework-Level instrumentation |
+| WordPress      | 4.x, 5.x             | PHP 7+                     | Framework-Level instrumentation |
+| Zend Framework | 1.12                 | All supported PHP versions | Framework-Level instrumentation |
+| Zend Framework | 2.x                  | All supported PHP versions | Generic web tracing             |
+| Yii            | 1.1, 2.0             | All supported PHP versions | Framework-Level instrumentation |
+| Drupal         |                      | All supported PHP versions | Generic web tracing             |
+| Magento        | 1, 2                 | All supported PHP versions | Generic web tracing             |
+| Phalcon        | 1.3, 3.4             | All supported PHP versions | Generic web tracing             |
+| Slim           | 2.x, 3.x, 4.x        | All supported PHP versions | Framework-Level instrumentation |
+| Neos Flow      | 1.1                  | All supported PHP versions | Generic web tracing             |
+| FuelPHP        | 1.1                  | PHP 7+                     | Generic web tracing             |
```

Comment:
For consistency, let's change instances of `Framework-Level` to `Framework-level` in this table.

Reply:
Good catch: https://github.com/DataDog/documentation/pull/9874/commits/1c1be02182021888b6e3ce0265b2ab89dcb05f95